src/planner/operator/logical_copy_to_file.cpp

```diff
@@ -39,7 +39,7 @@ unique_ptr<LogicalOperator> LogicalCopyToFile::Deserialize(Deserializer &deseria
 	auto file_path = deserializer.ReadProperty<string>(200, "file_path");
 	auto use_tmp_file = deserializer.ReadProperty<bool>(201, "use_tmp_file");
 	auto filename_pattern = deserializer.ReadProperty<FilenamePattern>(202, "filename_pattern");
-	auto overwrite_or_ignore = deserializer.ReadProperty<bool>(203, "overwrite_or_ignore");
```

Comment:
Don't we need some way to map `true` to `CopyOverwriteMode::COPY_OVERWRITE_OR_IGNORE` for backwards compatibility?
Or is that not as important for LogicalCopyToFile?

Reply:
Booleans are serialized as single bytes - where 0 is false and 1 is true. The new enum is also serialized as a single byte (`CopyOverwriteMode  : uint8_t`), where `0` is `COPY_ERROR_ON_CONFLICT` and 1 is `COPY_OVERWRITE `. So this should be work correctly already.